src/libs/OptionsListUtils.ts

```diff
@@ -1819,6 +1820,7 @@ function getOptions(
         }
     }
 
+    optionsToExclude.push({login: currentUserLogin});
```

Comment:
Do not push to optionsToExclude. Instead create a new variable 
```js
const personalDetailsOptionsToExclude = [...optionsToExclude, {login: currentUserLogin}];
```

Define this inside the `if (includePersonalDetails) { ... }` block

Reply:
Also remove this comment https://github.com/Expensify/App/blob/f60a61093e447477b14b65a5ef96caa3df36bbb6/src/libs/OptionsListUtils.ts#L1744